src/atomate2/lobster/schemas.py

```diff
@@ -301,7 +318,196 @@ def from_directory(
                 sb_icoop,
             )
         except ValueError:
-            return (None, None, None, None, None)
+            return None, None, None, None, None
+
+
+class DosComparisons(BaseModel):
+    """Model describing the DOS comparisons field in the CalcQualitySummary model."""
+
+    tanimoto_orb_s: Optional[float] = Field(
+        None,
+        description="Tanimoto similarity index between s orbital of "
+        "VASP and LOBSTER DOS",
+    )
+    tanimoto_orb_p: Optional[float] = Field(
+        None,
+        description="Tanimoto similarity index between p orbital of "
+        "VASP and LOBSTER DOS",
+    )
+    tanimoto_orb_d: Optional[float] = Field(
+        None,
+        description="Tanimoto similarity index between d orbital of "
+        "VASP and LOBSTER DOS",
+    )
+    tanimoto_orb_f: Optional[float] = Field(
+        None,
+        description="Tanimoto similarity index between f orbital of "
+        "VASP and LOBSTER DOS",
+    )
+    tanimoto_summed: float = Field(
+        None,
+        description="Tanimoto similarity index for summed PDOS between "
+        "VASP and LOBSTER",
+    )
+    e_range: list[float] = Field(
+        None,
+        description="Energy range used for evaluating the Tanimoto similarity index",
+    )
+    n_bins: int = Field(
+        None,
+        description="Number of bins used for discretizing the VASP and LOBSTER PDOS"
+        "(Affects the Tanimoto similarity index)",
+    )
+
+
+class ChargeComparisons(BaseModel):
+    """Model describing the charges field in the CalcQualitySummary model."""
+
+    BVA_Mulliken_agree: Optional[bool] = Field(
+        None,
+        description="Bool indicating whether atoms classification as cation "
+        "or anion based on Mulliken charge signs of LOBSTER "
+        "agree with BVA analysis",
+    )
+    BVA_Loewdin_agree: Optional[bool] = Field(
+        None,
+        description="Bool indicating whether atoms classification as cations "
+        "or anions based on Loewdin charge signs of LOBSTER "
+        "agree with BVA analysis",
+    )
+
+
+class BandOverlapsComparisons(BaseModel):
+    """Model describing the Band overlaps field in the CalcQualitySummary model."""
+
+    file_exists: bool = Field(
```

Comment:
Any field like this with a default value of None should probably be made Optional, particularly in support of pydantic2. If you don't want it optional, then I would set it to ... if you also don't know what the default value should be.

Reply:
Done, adapted the schema now